Attendance.csv

```diff
@@ -1,2 +1,2 @@
 Date,Session,Attendance,Notes,
-2017-01-11,Intro Git,x,,
+2017-01-11,Intro Git,9,3 physics 2 biomed engineering 1 undergrad math 1 psychology and Luke,
```

Comment:
The total is unchanged though, I think I counted her but didn't put her in the description.